decentraland/components/presentational/MyProfileCreationStatusItem.js

```diff
@@ -56,12 +57,14 @@ MyProfileCreationStatusItem.propTypes = {
 
 const styles = StyleSheet.create({
   container: {
+    flexWrap: "wrap",
     flexDirection: "row",
     backgroundColor: Colors.backgroundColor,
-    alignItems: "flex-start",
+    alignItems: "center",
     justifyContent: "flex-start",
-    marginTop: responsiveHeight(1),
-    height: responsiveHeight(6),
+    marginTop: responsiveHeight(2),
+    // height: responsiveHeight(FONT_SIZE),
```

Comment:
@marcelomorgado Happy to remove this before merging if you're happy with this change

Reply:
```suggestion
```